playbooks/roles/forum/templates/forum-watchdog.sh.j2

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+{% if ! devstack %}
+  tail -f {{ supervisor_log_dir }}/forum-stderr.log |
+  grep --line-buffered 'Mongo::Error::OperationFailure - not master and slaveOk=false (13435)' |
+  while read;
+  do
+    dt=`date '+%d/%m/%Y %H:%M:%S'`
+    echo "[$dt] Forum Failure detected - Restarting forum service..."
+    {{ supervisor_ctl }} restart forum
+  done
+{ % else %}
+  # On devstack just run a dummy loop to keep supervisor from trying to
+  # restart service
+  while true; do sleep 1d; done
```

Comment:
Couldn't we just omit this in `playbooks/roles/forum/tasks/deploy.yml` while running on devstack?

Reply:
What do you mean by this?

Reply:
I meant that we could omit creating watchdog at all if we are running devstack (instead of creating sleeping script).

Reply:
That's a better option, but I don't know how to do that :/

Reply:
Have you tried adding `when: not devstack` to the tasks?